mitmproxy/addons/tutorialapp/templates/mitmproxy/index.html

```diff
@@ -0,0 +1,210 @@
+{% extends "layout.html" %}
+{% block content %}
+<div class="row">
+    <nav class="col-md-2 bg-light sidebar sidebar-toc">
+        {% include 'mitmproxy/toc.html' %}
+    </nav>
+
+    <main role="main" class="col-md-6 ml-sm-auto px-md-4">
+        <h1 class="pt-3 pb-3 mb-3 border-bottom">mitmproxy Tutorial</h1>
+
+        <h2 id="overview" class="pt-3 pb-3 mb-3 border-bottom">Overview</h2>
+
+        <p>
+            This tutorial interactively introduces you to mitmproxy.
+            It will get you up to speed by teaching you the most common use-cases.
+            It will take approximately 10 minutes to complete, assuming you are comfortable on the command-line and familiar with HTTP.
+            <!-- You will learn how to perform common use-cases -->
+            <!-- Its purpose is to familiarize you with the most common use-cases. -->
+            <!-- It should get you up to speed with mitmproxy and provides sufficient  -->
+            <!-- Its meant as a quick-start guide, before diving deeper  -->
+            By the end of this tutorial, you will know how to:
+        </p>
+
+        <ul>
+            <li>Navigate mitmproxy's UI using your keyboard.</li>
+            <li>View HTTP flows and their details, such as HTTP headers.</li>
+            <li>Intercept and modify requests, before passing them to the server.</li>
+            <li>Replay previous requests, i.e., send them to the server again.</li>
+        </ul>
+
+        <p>
+            The tutorial is served by your local mitmproxy instance.
+            This implies that you have already set up your browser to use mitmproxy as a proxy server.
+            Thus, you have already completed the first step 🎉.
+        </p>
```

Comment:
Let's cut this paragraph. That is something we can list in next steps together with the next point, it doesn't matter for the tutorial itself.